### ISSUE TITLE
Can't setup local development environment

### ISSUE THREAD

**Issue by ddries** [electron-userland/electron-builder#6943]
* **Electron-Builder Version**: Latest `master` branch

* **Node Version**: v16.15.0

* **Electron Version**: v18.1.0
* **Electron Type (current, beta, nightly)**: current

* **Target**:  Windows x64

I've been trying to setup a local development environment in order to make some little contributions to the project. However, when I try to clone the repository and follow the guidelines in CONTRIBUTING.md, just after I try to build my test application with the local (`yalc`) electron-builder version, it raises the following error:

![image](https://user-images.githubusercontent.com/13487667/173673182-5cc9dc93-fe0a-40c4-9645-8c57723dd304.png)

Furthermore, when I try to run the tests in electron-builder clone with `pnpm test`, there are lots of tests which are not passed. It triggered my attention, since the repository is cloned and unmodified.

![image](https://user-images.githubusercontent.com/13487667/173673458-ac54e647-7c26-4843-b3e3-421d7ea01a1b.png)

I am following the guidelines in CONTRIBUTING.md, nothing else. Any clue why is this happening?

**Comment by mmaietta** [comment]
The snapshots are OS-specific to the build nodes they're running on. There isn't a way to make it platform-agnostic unfortunately. What I would recommend is using `TEST_FILES` env var for specifying the test you want to run. You can also execute them in the electron-builder docker image using `test-linux`

**Comment by ddries** [comment]
> The snapshots are OS-specific to the build nodes they're running on. There isn't a way to make it platform-agnostic unfortunately. What I would recommend is using `TEST_FILES` env var for specifying the test you want to run. You can also execute them in the electron-builder docker image using `test-linux`
> 
> Re: electron-updater import, can you confirm that the yalc version is pushed for every package in the monorepo?

Yep, I ran `yalc push` from every package is monorepo packages/ and the `yalc link` in the project depending on electron-builder.

I get that the best option to run the tests is building a docker image and running there the tests script, but how am I supposed to try the changes in my local environment if the imports are not working properly?

**Comment by ddries** [comment]
Closing since it probably won't ever get anywhere.

**Comment by mmaietta** [comment]
I'm not sure what to tell you. I just tried setting up a fresh project utilizing electron-quick-start-typescript with `import { autoUpdater } from 'electron-updater';` as the 3rd import and all yalc dependencies linked. It compiles just fine :/
